linux_os/guide/system/software/integrity/endpoint_security_software/mcafee_security_software/mcafee_endpoint_security_software/package_mcafeetp_installed/rule.yml

```
@@ -0,0 +1,45 @@
+documentation_complete: true
+
+prodtype: rhel7,rhel8
+
+title: 'Install McAfee Endpoint Security for Linux (ENSL)'
+
+description: |-
+    Install McAfee Endpoint Security for Linux antivirus software
+    which is provided for DoD systems and uses signatures to search for the
+    presence of viruses on the filesystem.
+
+    {{{ describe_package_install(package="mcafeetp") }}}
+
+rationale: |-
+    Virus scanning software can be used to detect if a system has been compromised by
+    computer viruses, as well as to limit their spread to other systems.
+
+severity: high
+
+identifiers:
+    cce@rhel7: CCE-86257-3
+    cce@rhel8: CCE-86260-7
+
+references:
+    disa: CCI-001233
+    nist: SI-2(2)
+    srg: SRG-OS-000191-GPOS-00080
+    stigid@rhel7: RHEL-07-020019
+    stigid@rhel8: RHEL-08-010001
+
+ocil_clause: 'the package is not installed'
+
+ocil: '{{{ ocil_package(package="mcafeetp") }}}'
+
+warnings:
+    - general: |-
+        Due to McAfee Endpoint Security for Linux (ENSL) being 3rd party software,
+        automated remediation is not available for this configuration check.
+
+platform: machine
+
+template:
+    name: package_installed
+    vars:
+        pkgname: mcafeetp
```

```suggestion
        pkgname: mcafeetp
    backends:
        bash: "off"
        ansible: "off"
        anaconda: "off"
        puppet: "off"
        blueprint: "off"
```

let's disable the remediation completely.